### DISCUSSION TITLE
How to let Output Message as an input Message and send it?

### DISCUSSION THREAD

**Discussion by yanyuxuanz** [gotd/td#781] [Q&A]
When I get updates.Message ,I want to Wrap this message to a messageRequest Input ,and Send it.  for example: How can I convert a MessageMediaClass to a InputMediaClass ,then send it ... Please,Thanks a lot！

**Comment by luomoxu** [comment]
https://github.com/iyear/tdl/blob/master/pkg/tmedia/convert.go
